src/FM.LiveSwitch.Hammer/Program.cs

```diff
@@ -45,15 +45,15 @@ static void Main(string[] args)
             result.MapResult(
                 (ClusterTestOptions options) =>
                 {
-                    return Run(new ClusterTest(options).Run).GetAwaiter().GetResult();
+                    return Run(new ClusterTest(options).Run, options).GetAwaiter().GetResult();
                 },
                 (LoadTestOptions options) =>
                 {
-                    return Run(new LoadTest(options).Run).GetAwaiter().GetResult();
+                    return Run(new LoadTest(options).Run, options).GetAwaiter().GetResult();
                 },
                 (ScanTestOptions options) =>
                 {
-                    return Run(new ScanTest(options).Run).GetAwaiter().GetResult();
+                    return Run(new ScanTest(options).Run, options).GetAwaiter().GetResult();
```

Comment:
Passing in `options` twice here smells. A better approach might be to have `ClusterTest`, `LoadTest`, and `ScanTest` all inherit from a base class (i.e. `Test`) that can process the `LogLevel` in its constructor.